ios/RNBackgroundFetch/RNBackgroundFetch.h

```diff
@@ -10,13 +10,8 @@
 #import <Foundation/Foundation.h>
 #import <UIKit/UIKit.h>
 
-#if __has_include("RCTEventEmitter.h")
-#import "RCTEventEmitter.h"
-#import "RCTLog.h"
-#else
-#import <React/RCTEventEmitter.h>
 #import <React/RCTLog.h>
-#endif
+#import <React/RCTAppearance.h>
```

Comment:
@mikehardy ?

Reply:
This is really vague, but I think the conditional import stuff is no longer needed as the header files moved around a long time ago (again, assuming I remember correctly) so that part seems fine, but I also don't see why you'd want to depend on Appearance vs EventEmitter. This is about events, not about handling contrast modes or dark mode

It looks like Appearance has actually been there since react-native 0.62, and I personally wouldn't support less than 0.64 at this point as the tail end of the version window, but also stated policy on version support window is different from actually cutting people off when not technically necessary (and without putting a version support / sunset policy in effect for one release cycle here) https://github.com/facebook/react-native/commit/63fa3f21c5ab308def450bffb22054241a8842ef

Reply:
Le me try not ussing Appearence